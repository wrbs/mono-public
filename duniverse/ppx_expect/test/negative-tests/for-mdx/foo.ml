(* $MDX part-begin=addition *)
open! Core

let%expect_test "addition" =
  printf "%d" (1 + 2);
  [%expect {| 4 |}]
;;

(* $MDX part-end *)
