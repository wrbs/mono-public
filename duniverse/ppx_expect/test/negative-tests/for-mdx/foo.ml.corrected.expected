(* $MDX part-begin=addition *)
open! Core

let%expect_test "addition" =
  printf "%d" (1 + 2);
  [%expect {| 3 |}]
;;

(* $MDX part-end *)
